packages/xyzservices/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: xyzservices
```

Comment:
Could you add some basic tests for xyzservices?

Reply:
Yes, no problem. I'll add them to the PR.